data/modules/CargoRun/CargoRun.lua

```diff
@@ -0,0 +1,1138 @@
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+explosives = Equipment.EquipType.New({
+	l10n_key = 'EXPLOSIVES', slots="cargo", price=50,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+furniture = Equipment.EquipType.New({
+	l10n_key = 'FURNITURE', slots="cargo", price=15,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+greenhouses = Equipment.EquipType.New({
+	l10n_key = 'GREENHOUSES', slots="cargo", price=20,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+hazardous_substances = Equipment.EquipType.New({
+	l10n_key = 'HAZARDOUS_SUBSTANCES', slots="cargo", price=100,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+machine_tools = Equipment.EquipType.New({
+	l10n_key = 'MACHINE_TOOLS', slots="cargo", price=10,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+neptunium = Equipment.EquipType.New({
+	l10n_key = 'NEPTUNIUM', slots="cargo", price=200,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+plutonium = Equipment.EquipType.New({
+	l10n_key = 'PLUTONIUM', slots="cargo", price=200,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+semi_finished_products = Equipment.EquipType.New({
+	l10n_key = 'SEMI_FINISHED_PRODUCTS', slots="cargo", price=10,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+spaceship_parts = Equipment.EquipType.New({
+	l10n_key = 'SPACESHIP_PARTS', slots="cargo", price=250,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+titanium = Equipment.EquipType.New({
+	l10n_key = 'TITANIUM', slots="cargo", price=150,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+tungsten = Equipment.EquipType.New({
+	l10n_key = 'TUNGSTEN', slots="cargo", price=125,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+uranium = Equipment.EquipType.New({
+	l10n_key = 'URANIUM', slots="cargo", price=175,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+quibbles = Equipment.EquipType.New({
+	l10n_key = 'QUIBBLES', slots="cargo", price=1,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+wedding_dresses = Equipment.EquipType.New({
+	l10n_key = 'WEDDING_DRESSES', slots="cargo", price=15,
+	capabilities={mass=1},
+	purchasable=false, icon_name="Default",
+	l10n_resource="module-cargorun"
+})
+
+local chemical = {}
```

Comment:
Why do you declare the equipments above as globals (baaaaaaaaad!), just to stash them in a local table afterwards ?

You could directly declare the table with the equipment inline, thus sparing the ~50 lines of table.insert